handlers/kinetic_request_ce_submission_clone_v1/handler/init.rb

```diff
@@ -74,12 +81,12 @@ def execute
         json[:coreState] = @parameters["state"] if !@parameters["state"].empty?
         json[:origin] = {"id" => @parameters["origin_id"]} if !@parameters["origin_id"].empty?
         json[:parent] = {"id" => @parameters["parent_id"]} if !@parameters["parent_id"].empty?
-        json[:values] = origValues.empty? ? {} : origValues
+        json[:values] = orig_values.empty? ? {} : orig_values
```

Comment:
This is such a weird pattern:

```
data = {}
data.tap do |json|
  ...
end
```

Why not just build the hash when creating it?
```
data = {
  :currentPage => {...},
  :coreState => ...
}
```